npm.js

```diff
@@ -62,12 +62,16 @@ exports.translate = function(load){
 				packages[pkg.name+"@"+pkg.version] = true;
 			}
 		});
-		var configDependencies = ['@loader','npm-extension'].concat(configDeps.call(loader, pkg));
+		var configDependencies = [
+			'@loader',
+			'npm-extension',
+			'process'
```

Comment:
I'm confused. Are you implying that the `process` dependency should be loaded before `npm-extension`? Or that the fact that `process` needs to be required at all is a non-starter?

It's not obvious to me why we'd opt-out of being compatible when we already know that some npm packages use Node's global `process` object. Why wouldn't we support those packages if a comprehensive list of requirements is available and a proven implementation is accessible?